### DISCUSSION TITLE
Tooltips with latex

### DISCUSSION THREAD

**Discussion by SPyofgame** [facelessuser/pymdown-extensions#1993] [Q&A] [answered]
I would like to inquire if it is possible to incorporate some advanced LaTeX formatting, such as mathematical equations and computer science notation, to enhance the quality of my content. For example, hinting my viewers whenever they hover over a weirdly defined symbol. Is there a way to achieve this without relying on the use of annotations or similar techniques ?

**Comment by facelessuser** [comment, marked as answer]
I'm not entirely sure what is being asked, is https://facelessuser.github.io/pymdown-extensions/extensions/arithmatex/ not enough? Or are you looking for something different? Maybe I need to see some examples.

**Reply by SPyofgame** [reply]
![image](https://user-images.githubusercontent.com/62207095/226241476-36f7ee75-7078-4796-86e4-399b330d5d81.png)

```cpp
[Bitmask][Reference]

[Reference]: https://github.com/ "Bitmask use bit $0, 1$ to represent a set of number, where integer $p$ is included in the set when and only when the $p$ bit of the bitmask is set ($1$)."
```

For something like that, can I use latex ?

And by the way, may I ask that are you and [squidfunk](https://github.com/squidfunk) both working on the same pymdown project ?

**Reply by facelessuser** [reply]
You can extend the MathJax config to transform content to math in all sorts of elements, but it doesn't naturally look in HTML title attributes (or any attributes really). You'd have to add some special code to accomplish that. [Here is an example](https://stackoverflow.com/a/40404772/3609487). The answer is from 2016, so the approach may have changed since then, but the short answer is that you need to code up a solution, or maybe squidfunk, if interested, could add support in his MkDocs theme.

> And by the way, may I ask that are you and [squidfunk](https://github.com/squidfunk) both working on the same pymdown project ?

Squidfunk supports MkDocs Material, but it relies on my extensions here for certain features. He does help sponsor this project, but our too projects are separate. I have a vested interest in MkDocs Material though as I rely on it for many of my code project's documentation.

**Reply by SPyofgame** [reply]
Should I also request this feature to [squidfunk](https://github.com/squidfunk) too ?

**Reply by facelessuser** [reply]
I guess if you really want such a feature, you could ask and see if he's open to the idea. With that said, tooltips do not currently support other rich content, it just takes plain text right now, so it may not be accepted.

**Reply by SPyofgame** [reply]
> You'd have to add some special code to accomplish that

Oh no, I don't think I am that good at web coding to fix the issues for incompatibilities or work case stuff when I try to implement that. 😞 

> He does help sponsor this project, but our too projects are separate. 

So that means there are some extra features in yours that I might be able to use too right ?🥇 

> I guess if you really want such a feature, you could ask and see if he's open to the idea

So I just open that request [here  🔗 ](https://github.com/squidfunk/mkdocs-material/discussions/5231)

Should I also close this discussion ? Thanks for your help

**Reply by facelessuser** [reply]
It looks like squidfunk answered your reply, so I will go ahead and consider this question answered.

In short, there are ways to do this, but it requires additional work, and since there are no plans to do so by me or the squidfunk, you would have to find some other party to implement this or do so yourself. 

I understand though that this may or may not be more complicated than you can handle, unfortunately, it is beyond the scope of work that I am currently willing to support.

**Reply by SPyofgame** [reply]
Nah, thank you for your support 👍 

Certain disadvantages like this make me appreciate more what I don't have and utilize those I have more suitable for the viewers & readers
